### ISSUE TITLE
Dashboard: Manual testing using keyboard and screen reader

### ISSUE THREAD

**Issue by swissspidy** [GoogleForCreators/web-stories-wp#4017]
## Task Description

The editor and the dashboard need to be usable via keyboard and have sufficient screen reader support. In a first step, we need to thoroughly test the current behavior. This way, we can uncover many low-hanging fruits that we can subsequently address.

We want to fix the biggest issues in this area so that we can kick-off a full accessibility review afterwards that will actually result in useful feedback. As long as we have many well-known issues, such a review does not make sense though.

Note: Testing should be done by engineers as they have the necessary context for how the app is built and which issues are to be expected.

**Comment by swissspidy** [comment]
Some resources from the [doc](https://docs.google.com/document/d/1aG9cDFxwkMfd8GK6obVATmxj3ZyahkFoCgffIMXWHIk/edit#):

[VoiceOver user guide](https://help.apple.com/voiceover/mac/10.15/) (macOS)
[ChromeVox user guide](https://support.google.com/accessibility/answer/7031755?hl=en) (Chromebook)
[ChromeVox Classic browser extension](https://chrome.google.com/webstore/detail/chromevox-classic-extensi/kgejglhpjiefppelpmljglcjbhoiplfn?hl=en) (Google Chrome)
[Web Content Accessibility Guidelines](https://www.w3.org/TR/WCAG/) (WCAG) 2.1
[Accessibility Developer Guide](https://www.accessibility-developer-guide.com/)
[Lighthouse Accessibility Audit](https://developers.google.com/web/tools/chrome-devtools/accessibility/reference#audits)

**Comment by swissspidy** [comment]
Refer to #4018 for resources and more infos on screen reader usage

**Comment by carloskelly13** [comment]
@BrittanyIRL has started work on the keyboard audit of implemented views on Dashboard.

https://docs.google.com/spreadsheets/d/1C4rpIt9KjFDGbdJHzE7xVdBXeAejHwu4ST_-6G2glJU/edit#gid=0

**Comment by swissspidy** [comment]
@carlos-kelly @BrittanyIRL I see a number of A11y tickets have been created for dashboard. Does that mean all testing using keyboard and screen reader has been completed?

Also, mind adding them to the #117 epic so we have a clearer picture of a11y-related workload?

Thanks!

**Comment by BrittanyIRL** [comment]
Hey, @swissspidy - the tickets made are to address p0 keyboard usability. Once that's squared away we'll do a more detail run through for screen readers. We wanted to silence any redundant noise from auditing with the known keyboard issues surrounding the grids.

**Comment by BrittanyIRL** [comment]
Compiled findings from auditing with chrome vox, Mac voice over, axe and lighthouse. We will create new tickets for findings. Got a 100% score on lighthouse for accessibility though! Closing since testing is complete.
